integration_tests/sdk/aqueduct_tests/test_functions/simple/model.py

```diff
@@ -1,31 +1,64 @@
+import pandas as pd
+
 from aqueduct import op
 
 
 @op()
 def dummy_sentiment_model(df):
-    df["positivity"] = 123
+    if isinstance(df, pd.DataFrame):
+        df["positivity"] = 123
+    else:
+        from pyspark.sql.functions import lit
```

Comment:
Could you put a banner comment on top of this file explaining what the `lit` import does in the spark case? Not clear to me why it has to be a decimal value either.